### ISSUE TITLE
Not returning results from page.evaluate

### ISSUE THREAD

**Issue by pnoeric** [berstend/puppeteer-extra#195]
This very simple code works flawlessly with regular puppeteer:

```Javascript
const puppeteer = require( 'puppeteer' );

// example HTML code for us to walk
const html = `
<html> <ul>
  <li>
      <div> <span>Joe</span> </div>
  </li>
  <li>
      <div> <span>Bob</span> </div>
  </li>
</ul> </html>`;

(async function main() {
  try {
    const browser = await puppeteer.launch();
    const [page] = await browser.pages();

    // load HTML into browser
    await page.goto(`data:text/html,${html}`);

    // walk through HTML and get a list of names
    const arrayOfNames = await page.evaluate(() => {
      const res = [];
      const xpathList = document.evaluate(
        '//ul//li/div//span',
        document.body, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null,
      );

      for (let i = 0; i < xpathList.snapshotLength; i++) {
        res.push(xpathList.snapshotItem(i).innerText);
      }

      return res;
    });

    // that's it... output the names and get outta here
    console.log(arrayOfNames);
    await browser.close();

  } catch (err) { console.error(err); }
})();
```

The correct output is: `[ 'Joe', 'Bob' ]`

However, if you change the first line to `require ( 'puppeteer-extra' )`, it fails.

Is this a known bug?

**Comment by pnoeric** [comment]
Anyone? ;-) Is puppeteer-extra still being developed?

**Comment by brunogaspar** [comment]
Don't think it's a known bug, but i can confirm it's failing with puppeteer-extra.

**Comment by brunogaspar** [comment]
Ok, this should be quite trivial to address.

On the example above the `launch()` does not have anything being passed, this is what's making it break, so having `launch({})` works, which means that it should have a sensible default internally.

@berstend 

https://github.com/berstend/puppeteer-extra/blob/master/packages/puppeteer-extra/src/index.ts#L153

Having the options argument to have a default of `{}` there or on  line https://github.com/berstend/puppeteer-extra/blob/master/packages/puppeteer-extra/src/index.ts#L156

Either way should make it work.

**Comment by brunogaspar** [comment]
Fixed on #281
